Change 28615 by mhx@mhx-r2d2 on 2006/07/25 18:42:26

	Very recent development versions of gcc (4.2.0 20060715) seem
	to have switched from '<command line>' to '<command-line>'.

Affected files ...

... //depot/perl/makedepend.SH#35 edit

Differences ...

--- perl/makedepend.SH
+++ perl/makedepend.SH
@@ -167,6 +167,7 @@
             -e '/^#.*<builtin>/d' \
             -e '/^#.*<built-in>/d' \
             -e '/^#.*<command line>/d' \
+            -e '/^#.*<command-line>/d' \
 	    -e '/^#.*"-"/d' \
 	    -e '/: file path prefix .* never used$/d' \
 	    -e 's#\.[0-9][0-9]*\.c#'"$file.c#" \
